### PR TITLE
Add support for additional --interface options: "pcf8574" and "bitbang_6800"

### PR DESCRIPTION
**Pull request by rm-hull**

### CODE REVIEW

File: luma/core/cmdline.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def load_config(path):
     return args
 
 
-class make_serial(object):
+class make_interface(object):
```
Comment (thijstriemstra) [inline luma/core/cmdline.py:124]: this is a backward incompat change. Can you add a `make_serial = make_interface` with a comment it's deprecated?

Comment (rm-hull) [reply luma/core/cmdline.py:124]: done

**Review by thijstriemstra — APPROVED** [review]
Looks good to me but a new release of luma.lcd (and luma.oled?) needs to be released first for this work?

**Comment by thijstriemstra** [comment]
ps. travis build is failing

**Comment by rm-hull** [comment]
Don’t think a new release of oled or lcd packages are necessary. This is just about exposing the interfaces (in luma.core) as command line args. At most, it will require a bump in luma.examples only.

**Comment by rm-hull** [comment]
Btw, I think there is another PR in the making, to tidy up `serial_interface` into just `interface` in the device classes. I will retain the old parameter name but mark it as deprecated, but this could have quite a ripple effect through all the packages ...

**Review by thijstriemstra — APPROVED** [review]

**Pull request merged**